ipaserver/install/replication.py

```diff
@@ -75,6 +75,17 @@
                'internalModifiersName',
                'internalModifyTimestamp')
 
+# settings for cn=replica,cn=$DB,cn=mapping tree,cn=config
+# during replica installation
+REPLICA_CREATION_SETTINGS = {
+    "nsds5ReplicaReleaseTimeout": ["20"],
+    "nsds5ReplicaBackoffMax": ["3"],
+    "nsDS5ReplicaBindDnGroupCheckInterval": ["2"]
+}
+# after replica installation
+REPLICA_FINAL_SETTINGS = REPLICA_CREATION_SETTINGS.copy()
+REPLICA_FINAL_SETTINGS["nsDS5ReplicaBindDnGroupCheckInterval"] = ["60"]
+
```

Comment:
Regarding REPLICA_FINAL_SETTINGS, it reset those values when the command ipa-replica-install complete.
Could we have a scenario where two ipa-replica-install step on the toes of each other so that one sets the parallel install settings on master (20s, 3s, 2s) and the other reset the the master to final value (60) so the first one will do its installation with not tuned master.

Reply:
Also the final settings are
nsds5ReplicaReleaseTimeout: 60
nsds5ReplicaBackoffMax: 300 (or better remove this attribute)
nsDS5ReplicaBindDnGroupCheckInterval: 60

Reply:
Yes that make sense, IMHO it will work most of the time

Reply:
So for me the remaining concern is the 3 values to restore at the end of the installation (not only nsDS5ReplicaBindDnGroupCheckInterval)